packages/api/src/graphql/mutation.private.ts

```diff
@@ -425,9 +427,11 @@ export const GraphQLAdminMutation = new GraphQLObjectType<undefined, Context>({
         id: {type: GraphQLNonNull(GraphQLID)},
         input: {type: GraphQLNonNull(GraphQLUserInput)}
       },
-      resolve(root, {id, input}, {authenticate, dbAdapter}) {
+      async resolve(root, {id, input}, {authenticate, dbAdapter}) {
         const {roles} = authenticate()
         authorise(CanCreateUser, roles)
+        input.email = input.email.toLowerCase()
```

Comment:
Should we implement a migration to transform all existing mails?

Reply:
That was in the commit before that is already merged this is just a fix of missing parameters